DOCS/man/osc.rst

```diff
@@ -310,6 +310,14 @@ Configurable Options
 
     Show total time instead of time remaining
 
+``remaining_playtime``
+    Default: yes
+
+    Whether the time-remaining display takes speed into account,
+    `yes` - how much playback time remains
```

Comment:
RST syntax needs two backticks (same for `no`).

Please also add a comma at the end of this line (the line break is ignored - like in markdown).

Reply:
added two newlines, and seems fixed now. can you check again?